csharp.html.markdown

```diff
@@ -33,7 +36,7 @@ using System.Threading.Tasks;
 using System.IO;
 
 // defines scope to organize code into "packages"
-namespace Learning
+namespace LearningXInYMinutes.CSharp
```

Comment:
I wanted to show how multiple levels of nesting work for "packages".

Should I add a little bit about namespace scoping?